pkg/reporter/reporter.go

```diff
@@ -16,7 +18,10 @@ import (
 	"github.com/anchore/k8s-inventory/pkg/inventory"
 )
 
-const ReportAPIPath = "v1/enterprise/kubernetes-inventory"
+const ReportAPIPathV1 = "v1/enterprise/kubernetes-inventory"
+const ReportAPIPathV2 = "v2/kubernetes-inventory"
+
+var Version = 0
```

Comment:
the global `Version` var should probably not be exported since how it is set is strictly controlled by an unexported function here and access is uncoordinated. Probably should unexport the other constants as well, but consider that a  non-blocking comment.

Reply:
Done